sign-in.txt

```diff
@@ -1,6 +1,7 @@
 ﻿This is an incredibly useful text document.
 //TODO: Be sure you have created a feature branch from develop 
 //TODO: Please put your name and github username AS SHOWN below ... 
-  ... then commit, pull origin develop, merge develop into your branch, push origin feature, and review your pull request
-  
-- Brian Bansenauer, github:Bansenauer-Cascadia
```

Comment:
Kevin - please leave lines 4-6 as given in the develop branch and just add your name below mine using the same format.